packages/cozy-device-helper/src/flagship.ts

```diff
@@ -15,7 +15,11 @@ export interface FlagshipMetadata {
   version?: string
 }
 
+const getGlobalWindow = (): Window =>
+  typeof window !== 'undefined' ? window : undefined
+
 export const getFlagshipMetadata = (): FlagshipMetadata =>
-  window.cozy?.flagship || {}
+  getGlobalWindow()?.window.cozy?.flagship || {}
```

Comment:
`getGlobalWindow()?.window` is that correct?. It looks like it will try `window?.window`